File: r/raindrops/raindrops.R
raindrops <- function(number) {
  pling <- if(!(number %% 3)) 'Pling' else ''
  plang <- if(!(number %% 5)) 'Plang' else ''
  plong <- if(!(number %% 7)) 'Plong' else ''
  output <- paste0(pling, plang, plong)
  return (if(output == '') as.character(number) else output)
}

File: typescript/raindrops/raindrops.ts
export default class Raindrops {
	convert = (n: number): string => {
		let output = '';
		if (!(n % 3)) {
			output += 'Pling';
		}
		if (!(n % 5)) {
			output += 'Plang';
		}
		if (!(n % 7)) {
			output += 'Plong';
		}
		return output ? output : String(n);
	};
}
